catalog/nsili/catalog-nsili-source/src/main/java/org/codice/alliance/nsili/source/NsiliSource.java

```diff
@@ -271,10 +273,33 @@ private void initCorbaClient() {
         setFilterDelegate();
     }
 
+    private void getIorFileFromLocalDisk() {
+        try(InputStream inputStream = new FileInputStream(iorUrl)) {
+            iorString = IOUtils.toString(inputStream);
+        } catch (IOException e) {
+            LOGGER.error("{} : Unable to process IOR String.", id, e);
+        }
+
+        if (StringUtils.isNotBlank(iorString)) {
+            LOGGER.debug("{} : Successfully obtained IOR file from {}", getId(), iorUrl);
+        } else {
+            LOGGER.error("{} : Received an empty or null IOR String.", id);
+        }
+    }
+
     /**
      * Uses the SecureClientCxfFactory to obtain the IOR file from the provided URL via HTTP(S).
      */
     private void getIorFileFromSource() {
+        URL url;
+        try {
+            url = new URL(iorUrl);
+        } catch (MalformedURLException e) {
+            LOGGER.debug("Local File Specified : {}", iorUrl);
```

Comment:
if someone puts `file://path/to/file` that will be valid and this will not fail.  you need to check the scheme of the URL to determine how you want to retrieve it.